proj/src/main/webapp/script.js

```diff
@@ -117,15 +117,16 @@ function appendMessage(payload){
  */
 
 const CHAT_ID = '-MB0ycAOM8VGIXlev5u8'
-const PATH = '/chat/'+CHAT_ID+'/messages'; // can make this more detailed (for example add user ID)
+// TODO: not hardcode, use path to an actual tag
+const PATH = '/chat/test/'+CHAT_ID+'/messages';
```

Comment:
Actually, can you make a getPath function that takes a chat id as a function so that in the future when chat id isn't static, it will be easier to change the path.

Reply:
fixed!